appium/tests/specs/mock/inbox/CheckRemoteImageRendering.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { SplashScreen, SetupKeyScreen, MailFolderScreen, EmailScreen } from '../../../screenobjects/all-screens';
+import { CommonData } from '../../../data';
+import { MockApi } from 'api-mocks/mock';
+import { MockApiConfig } from 'api-mocks/mock-config';
+import { GoogleMockMessage } from 'api-mocks/apis/google/google-messages';
+
+describe('INBOX: ', () => {
+  it('check remote image rendering', async () => {
+    const sender = CommonData.remoteImageRendering.sender;
+    const subject = CommonData.remoteImageRendering.subject;
+    const message = CommonData.remoteImageRendering.message;
+    // const remoteContentBlockedMessage = '[remote content blocked for your privacy]';
+    const mockApi = new MockApi();
+
+    mockApi.fesConfig = MockApiConfig.defaultEnterpriseFesConfiguration;
+    mockApi.ekmConfig = MockApiConfig.defaultEnterpriseEkmConfiguration;
+    mockApi.addGoogleAccount('e2e.enterprise.test@flowcrypt.com', {
+      messages: [subject] as GoogleMockMessage[],
+    });
+
+    await mockApi.withMockedApis(async () => {
+      await SplashScreen.mockLogin();
+      await SetupKeyScreen.setPassPhrase();
+      await MailFolderScreen.checkInboxScreen();
+
+      await MailFolderScreen.clickOnEmailBySubject(subject);
+      await EmailScreen.checkOpenedEmail(sender, subject, message);
+
+      // TODO: Check if WKWebView content contains remote content blocked image content. Couldn't seem to find a way to check WKWebView content
+      // https://discuss.appium.io/t/appium-and-wkwebview/4769/10
+      // await EmailScreen.checkEmailText(remoteContentBlockedMessage);
+      // await EmailScreen.checkEmailText('[img]');
```

Comment:
I left this as a `TODO` as I couldn't find way to check WKWebView content in appium.
https://discuss.appium.io/t/appium-and-wkwebview/4769/10.

@sosnovsky Do you know the way to check appium content?

Reply:
Probably we should enable web views debugging as described in Appium's docs - https://github.com/appium/appium-xcuitest-driver/blob/master/docs/device-preparation.md#enable-web-testing-availability

Reply:
Hmmm.. enabled web views debugging by setting `isInspectable` to true but still email body text becomes null.
Digging deeper now

Reply:
Probably we can just replace this test with Swift unit test, which will check that `Core.parseDecryptMsg` adds `remote content is blocked` text and replaces remote images with links

Reply:
Might be better to add webview content check logic for the future testing I think.